### DISCUSSION TITLE
"Upload failed"

### DISCUSSION THREAD

**Discussion by apfatzgo** [mediacms-io/mediacms#596] [Q&A] [answered]
I tried uploading a video (4,5 GB, 4K, H.264).
After the upload finished ("Processing") I got the message "Upload failed". I already increased the max upload in the config.

Screenshots:
After upload:
<img width="1372" alt="Bildschirmfoto 2022-06-20 um 22 30 11" src="https://user-images.githubusercontent.com/78032754/174679306-07f0ae20-ee00-438b-a81a-a34d87a087a8.png">
After "Processing":
<img width="1347" alt="Bildschirmfoto 2022-06-20 um 22 33 51" src="https://user-images.githubusercontent.com/78032754/174679333-edc3aea9-d8f6-4a41-a6be-43c38a12d833.png">

Environment:
 - OS: Ubuntu 18.04 (running on Proxmox)
 - Installation method: single server install
 - Browser: Safari 15.5

Thank you!

**Comment by Mozitux** [comment, marked as answer]
Hello, I had the same issue on low specs server when media encoding is pending. Try to wait until the last encoding is done before uploading a new video if your server seem to be slow. 

You can check the resources' consumption with htop command on Linux server.

**Comment by mgogoulos** [comment]
The default upload size should be enough, if not you can increase it from https://github.com/mediacms-io/mediacms/blob/main/deploy/local_install/nginx.conf and restart nginx

Also try uploading a small file and make sure whether it uploads without issues - in this case the problem here can be narrowed down to some size limit. If the small file fails as well this shows that uploads fail in general and probably not related to the size limit
